website/docs/advanced-concepts/embed-appsmith-into-existing-application.md

```diff
@@ -152,3 +150,26 @@ You can remove the Appsmith topbar with page tabs by appending **`?embed=true`**
 
 
 ![Appsmith Embed with ?embed=true property](/img/embed=true.png)
+
+### Embedding private apps
+
+:::info
+Appsmith supports embedding private apps only on self-hosted [**Business Edition**](https://www.appsmith.com/pricing) instances.
+:::
+
+You can embed private Appsmith apps on your website or in an external application and authenticate users seamlessly through SSO. 
+
+**Prerequisites**
+* The Appsmith app and the parent app should be under sub-domains of the same domain. For eg. `appsmith.company.com` and `internal.company.com`.
+* You must configure the same SSO identity provider(IDP) on the Appsmith and the parent applications.
+
+To configure Appsmith to use SSO, you can append the below parameters (based on your SSO type) to the URL path of the app you're embedding.
+* `?ssoTrigger=oidc` for OIDC SSO.
```

Comment:
The embed URLs already have `?embed=true`, in that case, this should be`&ssoTrigger=oidc`. Should we keep this more generic?

Reply:
Ah ok got it. makes sense. I'll change this back.